x-pack/plugins/security_solution/server/lib/detection_engine/rules/types.ts

```diff
@@ -132,7 +133,7 @@ export interface IRuleStatusResponseAttributes {
   last_failure_message: LastFailureMessage | null | undefined;
   last_success_at: LastSuccessAt | null | undefined;
   last_success_message: LastSuccessMessage | null | undefined;
-  status: JobStatus | null | undefined;
```

Comment:
Can this lead to a merge conflict with Madi's PR?

Reply:
I've cherry-picked this PR in Madi's branch to check for possible merge conflicts, and it looks good. There were just a couple of trivial conflicts to resolve.